src/signal.rs

```diff
@@ -1351,3 +1351,39 @@ extern "C" fn tree_view_column_trampoline(this: *mut GtkTreeViewColumn,
         f: &Box<Fn(TreeViewColumn) + 'static>) {
     f(TreeViewColumn::wrap_pointer(this))
 }
+
+pub trait GLAreaSignals {
+    fn connect_rendered(&self, f: Fn(GLArea, gdk::GLContext) + 'static);
+    fn connect_resized(&self, f: Fn(GLArea, i32, i32) + 'static);
+}
+
+#[cfg(gtk_3_16)]
+mod gl_area {
+    impl GLAreaSignals for GLArea {
+        fn connect_rendered(&self, f: Fn(GLArea, gdk::GLContext) + 'static) {
+            unsafe {
+                let f: Box<Box<Fn(GLArea, gdk::GLContext) + 'static>> = Box::new(Box::new(f));
+                connect(self.unwrap_pointer() as *mut _,"rendered",
+                    transmute(gl_area_trampoline), into_raw(f) as *mut _)
+            }
+        }
+
+        fn connect_resized(&self, f: Fn(GLArea, i32, i32) + 'static) {
+            unsafe {
+                let f: Box<Box<Fn(GLArea, i32, i32) + 'static>> = Box::new(Box::new(f));
+                connect(self.unwrap_pointer() as *mut _,"rendered",
+                    transmute(gl_area_trampoline_res), into_raw(f) as *mut _)
+            }
+        }
+    }
+
+    extern "C" fn gl_area_trampoline(this: *mut GtkGLArea, context: *mut gdk::GdkGLContext,
+            f: &Box<Fn(GLArea, gdk::GLContext) + 'static>) {
+        f(GLArea::wrap_widget(this), gdk::GLContext::wrap(context))
```

Comment:
This should be
```
f(GLArea::wrap_widget(this), from_glib_none(context))
```
similar to https://github.com/GuillaumeGomez/gtk/blob/master/src/signal.rs#L586